packages/dapp-toolkit/src/modules/wallet-request/transport/connector-extension/connector-extension.module.ts

```diff
@@ -137,22 +146,18 @@ export const ConnectorExtensionModule = (input: {
   ): ResultAsync<unknown, SdkError> => {
     const cancelRequestSubject = new Subject<Err<never, SdkError>>()
 
+    const maybeResolved$ = from(
+      requestResolverModule.getWalletResponseById(
+        walletInteraction.interactionId,
+      ),
+    ).pipe(filter((result) => result.isOk() && !!result.value))
```

Comment:
if it's error does it mean wallet responded with e.g. `rejectedByUser`?

Reply:
depends on the error recived from the wallet